cisc/cisc.go

```diff
@@ -357,7 +358,20 @@ func scQrcode(c *cli.Context) error {
 		return errors.New("Please chose one of the existing skipchain-ids")
 	}
 	scid := []byte(id.ID)
-	str := fmt.Sprintf("cisc://%s/%x", id.Data.Roster.RandomServerIdentity().Address.NetworkAddress(),
+	address := strings.Split(id.Data.Roster.RandomServerIdentity().Address.NetworkAddress(), ":")
+
+	if address[0] == "localhost" && c.Bool("e") {
```

Comment:
Add comment:
```
// Get our local IP address - this can be different from the public IP
// address returned by a service like `whatsmyip`, because we're behind
// a router.
```